Resources/Catalog/FileCatalogClient.py

```diff
@@ -8,29 +8,37 @@
 from types import ListType, DictType
 import os
 from DIRAC import S_OK, S_ERROR
-from DIRAC.Core.Base.Client import Client
 from DIRAC.ConfigurationSystem.Client.Helpers.Registry import getVOMSAttributeForGroup, getDNForUsername
+from DIRAC.DataManagementSystem.Utilities.CatalogUtilities import checkCatalogArguments
+from DIRAC.Core.DISET.RPCClient import RPCClient
 
-class FileCatalogClient( Client ):
+class FileCatalogClient( object ):
   """ Client code to the DIRAC File Catalogue
   """
   def __init__( self, url = None, **kwargs ):
     """ Constructor function.
     """
-    Client.__init__( self, **kwargs )
-    self.setServer( 'DataManagement/FileCatalog' )
+    self.__kwargs = kwargs
+    self.server = 'DataManagement/FileCatalog'
     if url:
-      self.setServer( url )
+      self.server = url
     self.available = False
-#    res = self.isOK()
-#    if res['OK']:
-#      self.available = res['Value']
+    
+  def __getRPC( self, rpc = None, url = '', timeout = 600 ):
```

Comment:
This is a _trivial_ function and I better keep its definition together with the only code that uses it as it is rather than generalizing it elsewhere. For the reader of the code hopping to another location to just look up a couple of lines is more painful than the advantage of "not duplicating the code".